### DISCUSSION TITLE
Configuration file for all clients

### DISCUSSION THREAD

**Discussion by SamaelAmbrona** [microsoft/winget-cli#2279] [Q&A] [answered]
Hello everybody,

I am not sure if this is the right place to ask this so please, excuse me if it is not.

Is there an easy way of deploy Winget and a specific configuration file to all computers in the company?

Is there any use on deploying a specific configuration or is it better include whatever needs to be configured as a parameter?

Best regards

**Comment by denelon** [comment]
@SamaelAmbrona 

[Group Policy](https://techcommunity.microsoft.com/t5/windows-it-pro-blog/manage-windows-package-manager-with-group-policy/ba-p/2346322) has the ability to configure many settings. What types of additional settings would you be interested in?

**Reply by SamaelAmbrona** [reply]
Hi Denelon, thanks for your reply.

I meant the JSON file for the winget settings. I was wondering first, if it would be wise to deliver same configuration to every client where we are going to use winget and how to do that.

My goal is to use winget for deploy apps to computer with Intune

**Comment by denelon** [comment, marked as answer]
@SamaelAmbrona

We've been iterating on the settings file as we add new features and improve user experience. We're also actively working on:
* #215 

That's why I was asking about specific settings. Settings tend to be "per user" rather than "per machine". If they are defined by policy, then the client can honor them even when a new user logs onto a machine the first time, before they have the "App Installer" package provisioned. You would need to know which users "AppData" needed to get the settings file ahead of time to pass a settings file out.

**Reply by SamaelAmbrona** [reply]
Understood.

Thank you very much for the clarification.
